modules/openy_features/openy_media/modules/openy_media_image/openy_media_image.install

```diff
@@ -266,7 +266,7 @@ function openy_media_image_update_8010() {
  * Change media browsers for images.
  */
 function openy_media_image_update_8011() {
-  \Drupal::service('module_installer')->install(['media_library', 'media_directories']);
+  \Drupal::service('module_installer')->install(['openy', 'media_library', 'media_directories']);
```

Comment:
it's more safe to install every other module in a foreach loop.
By passing multiple to single install() method we are facing with memory issues